web/src/app/rotations/RotationAddUserDialog.js

```diff
@@ -12,66 +11,54 @@ const mutation = gql`
   }
 `
 
-export default class RotationAddUserDialog extends React.Component {
-  static propTypes = {
-    rotationID: p.string.isRequired,
-    userIDs: p.array.isRequired,
-    onClose: p.func.isRequired,
+const RotationAddUserDialog = (props) => {
+  const { rotationID, onClose } = props
+  const [value, setValue] = useState(null)
```

Comment:
```suggestion
  const [value, setValue] = useState({
    users: []
  })
```

It might be worth setting the defaultValue as the default state so we don't have to have the separate checks throughout